spec/functional/mixin/powershell_out_spec.rb

```diff
@@ -22,8 +22,16 @@
   include Chef::Mixin::PowershellOut
 
   describe "#powershell_out" do
-    it "runs a powershell command and collects stdout" do
-      expect(powershell_out("get-process").run_command.stdout).to match /Handles\s+NPM\(K\)\s+PM\(K\)\s+WS\(K\)\s+VM\(M\)\s+CPU\(s\)\s+Id\s+/
+    context "for windows version less than 10", windows_10_or_2016: false do
+      it "runs a powershell command and collects stdout" do
+        expect(powershell_out("get-process").run_command.stdout).to match /Handles\s+NPM\(K\)\s+PM\(K\)\s+WS\(K\)\s+VM\(M\)\s+CPU\(s\)\s+Id\s+/
+      end
+    end
+
+    context "for windows version greater than 10", windows_10_or_2016: true do
```

Comment:
this should probably be `windows_after_10_or_2016: true` and should be changed to something more like `Gem::Requirement.new(">= 10").satisfied_by?(Gem::Version(host_version))` (which handles `10.10 > 10.9` correctly, unlike `to_f`).